src/main/java/org/myrobotlab/i2c/I2CBus.java

```diff
@@ -195,4 +195,9 @@ public void removeListener(String localTopic, String otherService) {
 
   }
 
+  @Override

```

Comment:
Why override this here?  shouldn't we be using the impl from the base Service class?

Reply:
heh .. you missed that this does not extend from Service .. ie its not a real service, nor is TimeEncoder ...
both should be promoted to real service class (TimeEncoder should handle multiple requests for time encoding, similar to Pid) .. but not related to this pr